selftests/pre_release/jobs/timesensitive.py

```diff
@@ -14,11 +15,13 @@
     'run.references': [os.path.join(ROOT_DIR, 'selftests', 'unit'),
                        os.path.join(ROOT_DIR, 'selftests', 'functional')],
     'filter.by_tags.tags': ['parallel:1'],
+    'nrunner.status_server_uri': '127.0.0.1:9999',
     'nrunner.max_parallel_tasks': 1,
     }
 
 
 if __name__ == '__main__':
-    with Job(CONFIG) as j:
+    suite = TestSuite.from_config(CONFIG)
+    with Job(CONFIG, [suite]) as j:
```

Comment:
I guess `Job.from_config(CONFIG)` is not possible here? It seems the test suite and job use the same config.

Reply:
Yes, in this case they are using, we could have something like:

```python
with Job.from_config(CONFIG) as j:
```

Reply:
In this case it could be simpler to parse, as long as we get the same end behavior.

Reply:
Yes, when creating a job using from_config() and if there is no suite's config, this method then will create one test suite with the same config. `len(job.test_suites) == 1` in this case.